src/VisualStudio/Core/Def/Implementation/Utilities/VisualStudioNavigateToLinkService.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Editor;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Roslyn.Utilities;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.Utilities
+{
+    [ExportWorkspaceService(typeof(INavigateToLinkService), layer: ServiceLayer.Host)]
+    [Shared]
+    internal sealed class VisualStudioNavigateToLinkService : INavigateToLinkService
+    {
+        [ImportingConstructor]
+        [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
+        public VisualStudioNavigateToLinkService()
+        {
+        }
+
+        public Task<bool> TryNavigateToLinkAsync(Uri uri, CancellationToken cancellationToken)
+        {
+            if (!uri.IsAbsoluteUri)
+            {
+                return SpecializedTasks.False;
+            }
+
+            if (uri.Scheme != Uri.UriSchemeHttp && uri.Scheme != Uri.UriSchemeHttps)
+            {
+                return SpecializedTasks.False;
+            }
```

Comment:
interesting.  So file:// schemes are not supported?  That seems unfortunate.  Do we not expect people to have relative/absolute file:// uris?

Reply:
➡️ Will combine with the follow-up issue for https://github.com/dotnet/roslyn/pull/35667#discussion_r287450606.

Reply:
➡️ Filed follow-up issue #37502